test/e2e/util.go

```diff
@@ -3003,6 +3191,42 @@ func createBadGenericPodAW(context *context, name string) *arbv1.AppWrapper {
 
 	return appwrapper
 }
+
+func createBadGenericItemAW(context *context, name string) *arbv1.AppWrapper {
+	//rb := []byte(`""`)
+	var schedSpecMin int = 1
+
+	aw := &arbv1.AppWrapper{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: context.namespace,
+		},
+		Spec: arbv1.AppWrapperSpec{
+			SchedSpec: arbv1.SchedulingSpecTemplate{
+				MinAvailable: schedSpecMin,
+			},
+			AggrResources: arbv1.AppWrapperResourceList{
+				GenericItems: []arbv1.AppWrapperGenericResource{
+					{
+						ObjectMeta: metav1.ObjectMeta{
+							Name:      fmt.Sprintf("%s-%s", name, "item"),
+							Namespace: context.namespace,
+						},
+						// GenericTemplate: runtime.RawExtension{
```

Comment:
I get the point of this comment here and in line 3196. We want to mess the generic items so we commented them, but left the comment so we know why it happens. I would change line 3196 to `// rb := []byte(`""`)` so that it is consistent.
PS: I can't decide if this is what we should do or if it should be removed all together. I'll think about it, but you can keep it like that for now.